README.md

```diff
@@ -28,6 +28,7 @@ If you begin to rely on this kafka setup we recommend you fork, for example to e
 
 | tag   | k8s ≥ | highlights |
 | ----- | ------ | ---------- |
+| master | 1.9.4 | Required for read-only ConfigMaps, k8s 1.9.4+ |
```

Comment:
Quick note that this is the readonly configmap volume was  backported to 1.8.9 and 1.7.14 so each of those need master as well.

Reply:
Thanks again @rubbish. I have apparently not been paying enough attention. I'll merge this now.